frontend/vite.config.ts

```diff
@@ -1,7 +1,18 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
+import os from 'os'
 
-// https://vite.dev/config/
+const platform = os.platform()
+console.log("platofrom" , platform)
 export default defineConfig({
   plugins: [react()],
+  server: {
+    watch: {
+      usePolling: platform === 'win32' || platform === 'linux',
```

Comment:
```suggestion
      usePolling: os.platform() === 'win32',
```

Reply:
We wanted it on Windows only, didn't we?

Reply:
I saw in google we need in linux also

Reply:
I've made the changes based on https://vite.dev/config/server-options#server-watch

Could you provide your source?

Reply:
in console it is returning me linux

Reply:
Let's focus on this: does hot reload work for you if you remove the 'linux' platform (keeping 'win32' only)?

Reply:
No

Reply:
I see... As we run it in a container, the host OS is irrelevant here. Just submit it as always enabled (usePolling: true) and we'll take care of customization later when we have frontend env variables support.

Sorry for the hassle with this issue @Yashgupta9330